daliuge-engine/dlg/apps/plasmaflight.py

```diff
@@ -32,56 +32,74 @@
 
 
 class PlasmaFlightClient:
-    def __init__(self, socket: str, scheme: str = "grpc+tcp", connection_args={}):
+    """
+    Client for accessing plasma-backed arrow flight data server.
+    """
+    def __init__(self, socket: str, scheme: str = "grpc+tcp", connection_args: Optional[dict] = None):
```

Comment:
default argument `{}` is unsafe as it behaves like a singleton for all calls. instantiating inside the constructor this way removes the lint warning